libraries/AC_AutoTune/AC_AutoTune_Multi.cpp

```diff
@@ -904,40 +908,30 @@ void AC_AutoTune_Multi::updating_angle_p_down_all(AxisType test_axis)
 // set gains post tune for the tune type
 void AC_AutoTune_Multi::set_tuning_gains_with_backoff(AxisType test_axis)
 {
+    // ensure gain_backoff has not been set outside limits
+    gain_backoff.set_and_save_ifchanged(constrain_float(gain_backoff, 0.5f, 1.0f));
```

Comment:
Don't think you mean this. Think you mean:
```
gain_backoff.set_and_save_ifchanged(constrain_float(gain_backoff, 0.0f, 0.5f));
```

Reply:
Opps!